src/ngraph/runtime/cpu/pass/cpu_fusion.cpp

```diff
@@ -1237,7 +1237,11 @@ void ngraph::runtime::cpu::pass::CPUFusion::construct_bounded_relu()
     auto relu = std::make_shared<op::Relu>(relu_input);
     auto iconst1 = op::Constant::create(element::f32, Shape{}, {1});
     auto alpha = std::make_shared<pattern::op::Label>(iconst1);
-    auto min = std::make_shared<op::Minimum>(relu, alpha);
+    auto broadcast_pred = [](std::shared_ptr<Node> n) {
```

Comment:
use `has_class` https://github.com/NervanaSystems/ngraph/blob/d18a9faf531f4cf4027c76a4b0d62cae822f984c/src/ngraph/pass/algebraic_simplification.cpp#L87

Reply:
@Krovatkin just wondering is their a benefit using  `has_class` v/s ` lambda` function like above?

Reply:
readability 😄